src/components/Dashboard/index.js

```diff
@@ -39,6 +51,38 @@ const Dashboard = () => {
   const dispatch = useDispatch();
   const errorProp = useSelector(({ auth }) => auth.profile.error);
   const loadingProp = useSelector(({ auth }) => auth.profile.loading);
+
+  const [name, setName] = useState("");
+  const [nameValidateError, setNameValidateError] = useState(false);
+  const [nameValidateErrorMessage, setNameValidateErrorMessage] = useState("");
+  const [orgName, setOrgName] = useState("");
+  const [orgNameValidateError, setOrgNameValidateError] = useState(false);
+  const [
+    orgNameValidateErrorMessage,
+    setOrgNameValidateErrorMessage,
+  ] = useState("");
```

Comment:
Move to a single line

Reply:
This is from Prettier format on save. I will disable that and get them to a single line. Thank you for mentioning that.

Reply:
@shehand I have a quick question. What is the maximum length for a line we should follow for this project so that I can follow the rule when formatting the code?

Reply:
Normally it would be 120 characters. But it is OK to have more than 120 characters if it is readable in PR review mode and things like these would be much readable if they are in a single line. Thant's why I asked to move this to single line.

Reply:
@shehand Thank you